client/src/components/ElementEditor/Header.js

```diff
@@ -160,6 +161,7 @@ class Header extends Component {
           <h3 className={titleClasses}>{element.title || noTitle}</h3>
         </div>
         {!simple && <div className="element-editor-header__actions">
+          <ElementCustomActionsComponent {...this.props} />
```

Comment:
I found that there's likely a need for all the props as we don't know what this will be used for (in my case I just needed the element ID)